src/modules/colorPicker/ColorPickerUI/Views/MainView.xaml

```diff
@@ -23,72 +25,71 @@
             </Border.Effect>-->
 
             <Grid>
-                <!--only color format - one line -->
+                <!--  only color format - one line  -->
                 <Grid Margin="2" Visibility="{Binding ShowColorName, Converter={StaticResource bool2InvertedVisibilityConverter}}">
                     <Grid.ColumnDefinitions>
-                        <ColumnDefinition Width="36"/>
-                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="36" />
+                        <ColumnDefinition Width="Auto" />
                     </Grid.ColumnDefinitions>
 
-                    <Border Background="{Binding ColorBrush}"
-                    HorizontalAlignment="Stretch"
-                    VerticalAlignment="Stretch"
-                    BorderBrush="{DynamicResource WindowBorderBrush}"
-                    BorderThickness="1"
-                    CornerRadius="4"/>
+                    <Border
+                        HorizontalAlignment="Stretch"
+                        VerticalAlignment="Stretch"
+                        Background="{Binding ColorBrush}"
+                        BorderBrush="{DynamicResource SurfaceStrokeColorDefaultBrush}"
+                        BorderThickness="1"
+                        CornerRadius="{DynamicResource ControlCornerRadius}" />
 
-                    <TextBlock 
-                       x:Name="ColorTextBlock"
-                       Margin="8,5,8,8"
-                       FontSize="16"
-                       FontWeight="SemiBold"
-                       Foreground="{DynamicResource PrimaryForegroundBrush}"
-                       Grid.Column="1"
-                       AutomationProperties.LiveSetting="Assertive"
-                       AutomationProperties.LabeledBy="{Binding ColorName}" 
-                       AutomationProperties.Name="{Binding ColorName}" 
-                       Text="{Binding ColorText}"/>
+                    <TextBlock
+                        x:Name="ColorTextBlock"
+                        Grid.Column="1"
+                        Margin="8,5,8,8"
+                        AutomationProperties.LiveSetting="Assertive"
+                        AutomationProperties.Name="{Binding ColorName}"
+                        FontSize="16"
+                        FontWeight="SemiBold"
+                        Text="{Binding ColorText}" />
                 </Grid>
 
-                <!--color format and color name - two lines-->
+                <!--  color format and color name - two lines  -->
                 <Grid Margin="2" Visibility="{Binding ShowColorName, Converter={StaticResource bool2VisibilityConverter}}">
```

Comment:
`bool2VisibilityConverter` I'm sure we have these converters already in a ToolKit or something?

Reply:
From the Toolkit? This is a WPF app, we don't have access to the Toolkit converters here.